conf/app.routes

```diff
@@ -4,7 +4,11 @@ GET         /assets/*file                           controllers.Assets.versioned
 ->          /hmrc-frontend                          hmrcfrontend.Routes
 
 GET         /                                       controllers.CashAccountController.showAccountDetails(page: Option[Int])
+GET         /v2                                     controllers.CashAccountV2Controller.showAccountDetails(page: Option[Int])
```

Comment:
are we updating this URL later?

Reply:
Yes, design team will come up with the required url later. Its only a temp url. Once this feature is on production, users should use /